lib/hooks/ReturnStatement.js

```diff
@@ -42,7 +42,7 @@ exports.getIndentEdges = function(node, opts) {
   return parentheses ?
     {
       startToken: parentheses.opening,
-      endToken: parentheses.closing
+      endToken: parentheses.closing.prev
```

Comment:
thanks a lot for the feedback.. found a way to revert the old changes and to make the `indentInBetween` behave in a way that makes more sense to us.